usecases/stable/documentationcamera/recording.documentationcamera.realtime/odcockpit/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Dockerfile - Dockerfile to run OpenDLV software.
+# Copyright (C) 2016 Christian Berger
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Date: 2016-09-09
+
+FROM seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:latest
```

Comment:
You are right. Use cases with the latests tag should not be in the stable folder. But opendlv.core evolves really fast and sometimes in order to quickly test a new version, I have to manually change the tag in the docker-compose file from a fixed release to latest. In this way, I do not have to frequently move use cases between the stable folder and latest folder.

Maybe the easiest way is to replace the “latest" tag in the docker-compose file with the latest version number? Now the number of use cases is growing and re-testing existing use cases with new releases becomes more time-consuming. What do you suggest?